packages/client3/pages/artist/pool/[address]/pool-details-grid-right.tsx

```diff
@@ -18,27 +17,27 @@ type Props = {
 function PoolDetailsRightGrid({ poolData, tranchedPoolData }: Props) {
   const terms = poolData?.terms;
 
-  // Note: Need to use Borrower.sol to interact with TranchedPool.sol
   const borrowerContract = useContract("Borrower", poolData.borrowerContract);
 
-  const goldfinchFactory = useContract(
-    "GoldfinchFactory",
-    CONTRACT_ADDRESSES.GoldfinchFactory
+  const USDC = useContract(
+    "USDC",
+    "0x3E0B09aDf6171F5D1aefef567BA6Cf1fb364E080"
   );
 
-  const disabled = !(
-    tranchedPoolData.estimatedTotalAssets - tranchedPoolData.totalDeployed
+  const CreditLine = useContract(
+    "CreditLine",
+    "0x099758bd8b67658e153357d302da655c52b3aefd"
```

Comment:
Can this be a constant as well?

Reply:
Yeah I just saw this too. Nah I need to change it to the specific pool's creditline.

Reply:
Also variable could be camelCased